### DISCUSSION TITLE
Adding flags to startup script

### DISCUSSION THREAD

**Discussion by LukasOlivier** [AlexProgrammerDE/balena-minecraft-server#195] [Q&A] [answered]
Hi!
Is there a way to add flags to the startup script of the minecraft server?
I looked for the start.sh file containing `java -Xms{RAM}G -Xmx{RAM}G  -jar {JARFILE}.jar` but didn't find it.

Thanks in advance!

**Comment by AlexProgrammerDE** [comment]
What type of flags are you planning to add?

**Comment by viedok** [comment, marked as answer]
Hey, you can find the start command in _MC SERVER > start.sh_ on line _101_. When you download the files, just modify this line before deploying and you'll be good to go.
Alternatively, you can set custom flags with variables (Aikar's flags and ZGC are implemented right now).

**Reply by LukasOlivier** [reply]
Ahha, I did not notice that. That's all the flags I needed. Thank you for the explanation!

**Reply by viedok** [reply]
no problem :)
